src/summary/history/notes-card.component.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { match } from "react-router";
+
+import dayjs from "dayjs";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import { encounterResource } from "./encounter.resource";
+import NotesValues from "../cards/notes-values.component";
+import style from "./notes-card-style.css";
+import SummaryCardFooter from "../cards/summary-card-footer.component";
+
+export default function NotesCard(props: NotesCardProps) {
+  const [patientNotes, setPatientNotes] = React.useState(null);
+  const [showMore, setShowMore] = React.useState(false);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+
+    encounterResource(props.currentPatient.identifier[0].value, abortController)
+      .then(({ data }) => {
+        if (data.total > 0) {
+          setPatientNotes(getNotes(data.entry));
+        }
+      })
+      .catch(error => {

```

Comment:
This is incorrect usage of createErrorHandler:

```suggestion
      .catch(createErrorHandler())
```